README.md

```diff
@@ -8,45 +8,60 @@
-Windows due to limitations with our regridding software.
+</div>
 
-Note that ClimaLand.jl is a different model from the original CliMA Land,
-which aims to utilize remote sensing data through more complex canopy RT
-and plant physiology modules. For more details, please refer to
-https://github.com/CliMA/Land.
-- Wang, Yujie, et al. "Testing stomatal models at the stand level in deciduous angiosperm and evergreen gymnosperm forests using CliMA Land (v0. 1)." Geoscientific Model Development 14.11 (2021): 6741-6763.
-- R. K. Braghiere, Y. Wang, R. Doughty, D. Souza, T. Magney, J. Widlowski, M. Longo, A. Bloom, J. Worden, P. Gentine, and C. Frankenberg. 2021. Accounting for canopy structure improves hyperspectral radiative transfer and sun-induced chlorophyll fluorescence representations in a new generation Earth System model. Remote Sensing of Environment. 261: 112497.
-- Wang, Yujie, and Christian Frankenberg. "On the impact of canopy model complexity on simulated carbon, water, and solar-induced chlorophyll fluorescence fluxes." Biogeosciences 19.1 (2022): 29-45.
-- Wang, Yujie, et al. "GriddingMachine, a database and software for Earth system modeling at global and regional scales." Scientific data 9.1 (2022): 258.
-- Holtzman, Nataniel, et al. "Constraining plant hydraulics with microwave radiometry in a land surface model: Impacts of temporal resolution." Water Resources Research 59.11 (2023): e2023WR035481.
+# ClimaLand.jl
+
+This is the repository of the CliMA land model code. Here are some notable features:
+- ClimaLand has a modular design, models can be run as standalone (e.g., soil moisture only) or integrated (e.g., soil moisture and energy AND canopy AND snow, etc.)
+- ClimaLand is CPU and GPU compatible
+- ClimaLand welcome contributions: please feel free to reach out to us with questions about how to get started, create a branch, and extend our code. For example, a modeler might want to test a new stomatal conductance model. 
+- ClimaLand provides APIs and UIs at multiple levels. 
 
 ## Models
 
-Component Models:
-RichardsModel <: AbstractSoilModel <: AbstractImExModel <: AbstractModel [runnable w/o LandModel wrapper as well]
+In our code base, a "model" define a set of prognostic variables which must be timestepped. The equations which govern the time evolution likely contain parameters and are informed by parameterization and physical domain choices. Any ClimaLand model contains all of the information needed to evaluate these equations. Below are the current models we support:
 
-EnergyHydrologyModel <: AbstractSoilModel <: AbstractImExModel <: AbstractModel [runnable w/o LandModel wrapper as well]
+<strong> Component Models: </strong>
 
-CanopyModel <: AbstractVegetationModel <: AbstractExpModel <: AbstractModel  [runnable w/o LandModel wrapper as well]
+- RichardsModel <: AbstractSoilModel <: AbstractModel (runnable in standalone mode, or as part of a land model)
```

Comment:
just one small comment here - RichardsModel currently only runs in standalone. not sure if we would want it ever in a land model since it doesnt have temperature and phase change in soil?

Reply:
no you're right, we don't need it in land model. I'll update to "runnable only in standalone mode"